lua/autorun/client/cl_devtools.lua

```diff
@@ -1,144 +1,165 @@
-﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+﻿--[[
+    Limitations:
+        - If a path has lua/STRUCTHERE eg. "lua/autorun/lua/weapons/test.lua" then
+            It'll pass the Struct into it, not sure if it's a bug or a feature.
+        - A file within a sub directory of a struct folder like: "lua/weapons/testEnt/mysubdir/lua"
+            Won't have the SWEP/struct table passed to it.
+
+]]--
+
+-- This file is subject to copyright - contact swampservers@gmail.com for more information.
 -- INSTALL: CINEMA
 SWAMP_DEV = SWAMP_DEV or {}
-SWAMP_DEV.isDev = SWAMP_DEV.isDev
+-- Are we in development mode?
+SWAMP_DEV.isDev = false
```

Comment:
Could just do a nil check, no real need for the assignment to false here.

Reply:
Addressed in a more recent comment.